source/use-cases/local-master-key.txt

```diff
@@ -1,89 +1,23 @@
 A. Create a Local Master Key
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Client-Side Field Level Encryption for MongoDB uses the envelope encryption mechanism in which there are two types of keys: 1) **data encryption keys** which are used to encrypt/decrypt the sensitive data fields and 2) a **master key** that is used to encrypt the **data encryption keys**.
+Client-Side Field Level Encryption for MongoDB uses an encryption strategy called envelope encryption in which keys used to encrypt/decrypt data (called **data encryption keys**) are encrypted with another key (called the **master key**). 
 
-The **master key** is stored separately in a Key Management System (KMS) in order to prevent the **data encryption keys** from being used directly to access the sensitive data.
+This strategy bolsters the security, minimizes encryption/decryption overhead, and allows the data encryption keys to be stored in the same replica set as the encrypted data. If a basic encryption strategy was used instead of envelope encryption, the computational and network load of encryption and decryption would need to be handled by the system that manages your keys. For more information on envelope encryption and key management concepts, read `this AWS article <https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#enveloping>`_.
 
-The **data encryption keys** can safely be stored in a MongoDB collection alongside the encrypted sensitive data fields since they have been encrypted by the **master key**.
+The master key is stored separately in a Key Management System (KMS) in order to prevent the data encryption keys from being used directly to access the sensitive data.
 
-In Client-Side Field Level Encryption, a client that has access to both **master key** and **data encryption keys** is necessary to perform encryption/decryption.
+The data encryption keys can safely be stored in a MongoDB collection alongside the encrypted sensitive data fields since they have been encrypted by the master key.
 
-In the code example below, we will set up the **master key** in a local KMS store as a file. This setup should only be used for local testing purposes and never for production. A secure service such as AWS KMS should always be used in production environments to ensure the **master key** is not compromised.
+In Client-Side Field Level Encryption, a client that can access both master key and data encryption keys is necessary to perform encryption/decryption.
 
+In the steps below, we set up a local master key saved to a file and load it into the MongoDB client settings in your development environment.
```

Comment:
"set up a local master key" and "save to a file" should be distinct

- we generate a local master key
- save it to a file
- and configure a MongoDB client to use the key

Reply:
Good suggestion. I will need to defer on this until we figure out whether to include the configuration step.